src/components/pages/areas-manage/AreasManage.js

```diff
@@ -135,7 +135,7 @@ class AreasManage extends React.Component {
                     />
                 </div>
               </div>
-              <Button className="c-button">Save</Button>
+              <Button className="c-button" disabled={this.props.saving}>Save</Button>
```

Comment:
Its resolving on a bad request and redirecting to index without creating the area if we don't have a name